[2021-11-30 22:12:21,883] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: twitter_dag.twitter_aluraonline 2021-11-29T09:00:00+00:00 [queued]>
[2021-11-30 22:12:21,894] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: twitter_dag.twitter_aluraonline 2021-11-29T09:00:00+00:00 [queued]>
[2021-11-30 22:12:21,894] {taskinstance.py:880} INFO - 
--------------------------------------------------------------------------------
[2021-11-30 22:12:21,894] {taskinstance.py:881} INFO - Starting attempt 1 of 1
[2021-11-30 22:12:21,894] {taskinstance.py:882} INFO - 
--------------------------------------------------------------------------------
[2021-11-30 22:12:21,904] {taskinstance.py:901} INFO - Executing <Task(TwitterOperator): twitter_aluraonline> on 2021-11-29T09:00:00+00:00
[2021-11-30 22:12:21,906] {standard_task_runner.py:54} INFO - Started process 2084 to run task
[2021-11-30 22:12:21,982] {standard_task_runner.py:77} INFO - Running: ['airflow', 'run', 'twitter_dag', 'twitter_aluraonline', '2021-11-29T09:00:00+00:00', '--job_id', '32', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/twitter_dag.py', '--cfg_path', '/tmp/tmp51bx2skk']
[2021-11-30 22:12:21,982] {standard_task_runner.py:78} INFO - Job 32: Subtask twitter_aluraonline
[2021-11-30 22:12:22,007] {logging_mixin.py:112} INFO - Running <TaskInstance: twitter_dag.twitter_aluraonline 2021-11-29T09:00:00+00:00 [running]> on host kauai-VirtualBox
[2021-11-30 22:12:22,068] {base_hook.py:89} INFO - Using connection to: id: twitter_default. Host: https://api.twitter.com, Port: None, Schema: None, Login: None, Password: None, extra: XXXXXXXX
[2021-11-30 22:12:22,070] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2021-11-29T09:00:00.00Z&end_time=2021-11-30T09:00:00.00Z
[2021-11-30 22:12:22,337] {logging_mixin.py:112} WARNING - /home/kauai/datapipeline/venv/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2021-11-30 22:12:22,735] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2021-11-29T09:00:00.00Z&end_time=2021-11-30T09:00:00.00Z&next_token=b26v89c19zqg8o3fpdy7o6irkjqd6r4e7hgqjo8a6de9p
[2021-11-30 22:12:22,735] {logging_mixin.py:112} WARNING - /home/kauai/datapipeline/venv/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2021-11-30 22:12:23,111] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2021-11-29T09:00:00.00Z&end_time=2021-11-30T09:00:00.00Z&next_token=b26v89c19zqg8o3fpdy7o4g4y7tg30rb77ojbtw6s7qt9
[2021-11-30 22:12:23,111] {logging_mixin.py:112} WARNING - /home/kauai/datapipeline/venv/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2021-11-30 22:12:23,478] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2021-11-29T09:00:00.00Z&end_time=2021-11-30T09:00:00.00Z&next_token=b26v89c19zqg8o3fpdy7o4f8gc4d6jqerl884g6oege4d
[2021-11-30 22:12:23,479] {logging_mixin.py:112} WARNING - /home/kauai/datapipeline/venv/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2021-11-30 22:12:23,848] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2021-11-29T09:00:00.00Z&end_time=2021-11-30T09:00:00.00Z&next_token=b26v89c19zqg8o3fpdy7o4dqudpxhkr16c6bv85uq4o71
[2021-11-30 22:12:23,848] {logging_mixin.py:112} WARNING - /home/kauai/datapipeline/venv/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2021-11-30 22:12:24,180] {taskinstance.py:1057} INFO - Marking task as SUCCESS.dag_id=twitter_dag, task_id=twitter_aluraonline, execution_date=20211129T090000, start_date=20211201T011221, end_date=20211201T011224
[2021-11-30 22:12:26,878] {local_task_job.py:102} INFO - Task exited with return code 0
